apps/client/src/app/pages/resources/resources-page.module.ts

```diff
@@ -1,12 +1,19 @@
 import { CommonModule } from '@angular/common';
 import { CUSTOM_ELEMENTS_SCHEMA, NgModule } from '@angular/core';
+import { MatTabsModule } from '@angular/material/tabs';
+import { RouterModule } from '@angular/router';
 
 import { ResourcesPageRoutingModule } from './resources-page-routing.module';
 import { ResourcesPageComponent } from './resources-page.component';
 
 @NgModule({
   declarations: [ResourcesPageComponent],
-  imports: [CommonModule, ResourcesPageRoutingModule],
+  imports: [
```

Comment:
Please order imports alphabetically

Reply:
![image](https://github.com/user-attachments/assets/e0a5feab-2493-497a-9bc1-e35fa98016e7)
Is this alright?

Reply:
> Is this alright?

Yes 👍🏻